tests/sentry/api/endpoints/test_organization_profiling_profiles.py

```diff
@@ -808,6 +808,99 @@ def test_queries_profile_candidates_from_profiles_with_continuous_profiles_with_
                 },
             )
 
+    @patch("sentry.profiles.flamegraph.bulk_snuba_queries")
+    @patch("sentry.api.endpoints.organization_profiling_profiles.proxy_profiling_service")
+    def test_queries_profile_candidates_from_spans(
+        self,
+        mock_proxy_profiling_service,
+        mock_bulk_snuba_queries,
+    ):
+        # this span has a transaction profile
+        profile_id = uuid4().hex
+
+        span = self.create_span(project=self.project, start_ts=self.ten_mins_ago, duration=1000)
+        span.update({"profile_id": profile_id})
+
+        self.store_span(span, True)
```

Comment:
Nit: would be clearer if you passed it as `is_eap=True` to make it obvious what the 2nd arg is for. Same for the rest of the test